modules/ROOT/pages/index.adoc

```diff
@@ -1,27 +1,14 @@
-= Bonita Intelligent Continuous Improvement Add-on
+= Labs Program
 :page-aliases: release_notes.adoc
```

Comment:
You are using the same alias twice -> error

Reply:
removed. Thanks